study.md

```diff
@@ -21,18 +21,20 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Call-backs...not so when/why we use them.
```

Comment:
Here's a small bit of code that I wrote for demonstrating the use of callbacks.  Let me know if it's unclear.
```javascript
// This function has two parameters: greeter (a callback function) and name (a
// string).  It will call "greeter" with "name" as its argument.
function greet(greeter, name) {
  console.log(`Executing greeter callback with ${name} as the argument.`);

  greeter(name);

  console.log('Greeting finished.');
}

// A named function to be used as a callback function.
function hello(name) {
  console.log(`Hello, ${name}.`);
}

// Another named function to be used as a callback function.
function hi(name) {
  console.log(`Hi, ${name}.`);
}

// Calling greet() with hello() as its callback function.
greet(hello, 'Colin');
// Output:
// Executing greeter callback with Colin as the argument.
// Hello, Colin.
// Greeting finished.

// Calling greet() with hello() as its callback function.
greet(hi, 'Colin');
// Output:
// Executing greeter callback with Colin as the argument.
// Hi, Colin.
// Greeting finished.

// Calling greet() with an anonymous function as its callback function.
greet(someName => {
  console.log(`Greetings, ${someName}.`);
}, 'Colin');
// Output:
// Executing greeter callback with Colin as the argument.
// Greetings, Colin.
// Greeting finished.
```